modules/fuzzy/README.md

```diff
@@ -0,0 +1,5 @@
+Fuzzy image processing
+=======================
+
+This module is focused on the image processing using fuzzy mathematics, namely F-transform.
+
```

Comment:
It would be nice if you could update this documentation to contain a structure of the module with the corresponding functionality, which was the intention of adding those README files. Also you can use it to point people to your contact details for questions and usability of the module.